addons/website_forum/models/forum.py

```diff
@@ -150,6 +152,7 @@ def _compute_teaser(self):
                     forum.teaser = forum.description
             else:
                 forum.teaser = ""
+            forum.teaser_non_stored = forum.teaser
```

Comment:
-> use forum description as teaser will be stored with the incorrect value anyway